redisinsight/api/src/modules/database/database.service.spec.ts

```diff
@@ -117,6 +137,23 @@ describe('DatabaseService', () => {
     });
   });
 
+  describe('getWithoutSecurityFields', () => {
```

Comment:
lack of other hidden fields

Reply:
add test for ssh password and ssh passphrase (password was already implemented)